var/spack/repos/builtin/packages/protobuf/package.py

```diff
@@ -15,6 +15,7 @@ class Protobuf(Package):
     homepage = "https://developers.google.com/protocol-buffers"
     url = "https://github.com/protocolbuffers/protobuf/archive/v3.18.0.tar.gz"
 
+    version("21.7", sha256="75be42bd736f4df6d702a0e4e4d30de9ee40eac024c4b845d17ae4cc831fe4ae")
```

Comment:
@bvanessen it looks like you removed 21.1 in #32491, was there a reason for this?

Reply:
Because there is a really weird thing that they are doing is having equivalent versions between 21.5 and 21.7 which are equivalent to 3.21.5 and 3.21.7.  I thought that we should be consistent.  So this should be 3.21.7.

Reply:
Any idea why they're doing this?

Reply:
@iarspider can you replace this with 3.21.7?